WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsPluginNotActivatedView.swift

```diff
@@ -23,23 +25,23 @@ struct InPersonPaymentsPluginNotActivated: View {
 
 private enum Localization {
     static let title = NSLocalizedString(
-        "Activate WooCommerce Payments",
-        comment: "Title for the error screen when WooCommerce Payments is installed but not activated"
+        "Activate %@",
+        comment: "Title for the error screen when a Card Present Payments extension is installed but not activated"
     )
 
     static let message = NSLocalizedString(
-        "The WooCommerce Payments extension is installed on your store but not activated. Please activate it to accept In-Person Payments",
-        comment: "Error message when WooCommerce Payments is not activated"
+        "The %@ extension is installed on your store but not activated. Please activate it to accept In-Person Payments",
```

Comment:
Same comment about positional placeholders

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5935/commits/94aaad6168c39dd525bc63b2dba938d888860b3e